ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -3207,8 +3243,6 @@ def test_torch_instance_arccos_(
         method_flags=method_flags,
         frontend=frontend,
         on_device=on_device,
-    )
```

Comment:
Hi @vtsamit @soma2000-lang, please revert the changes here, it seems like you accidentally deleted the close bracket.

Reply:
Really sorry sir. Can this PR be opened once again so that I can add the close bracket?